Makefile

```diff
@@ -154,7 +154,7 @@ app-sre-saas-template: hypershift
 # Run tests
 .PHONY: test
 test:
-	$(GO) test -race -count=25 -timeout=30m ./... -coverprofile cover.out
+	$(GO) test -race -count=25 -timeout=30m $(shell go list ./... | grep -v e2e) -coverprofile cover.out
```

Comment:
nit: Not sure if we can change it, but it would be nice if the list of directories doesn't display when running `make test`.

Reply:
You can do that just adding the "@" in front of the "$(GO)" command:

```
@$(GO) test -race -count=25 -timeout=30m $(shell go list ./... | grep -v e2e) -coverprofile cover.out
```